docs/source/index.rst

```diff
@@ -17,7 +17,7 @@ However, most of the new data remain under-utilized.
 echopype aims to address the root cause of this problem - the lack of
 interoperable data format and scalable analysis workflows that adapt well
 with increasing data volume - by providing open-source tools as entry points for
-scientists to make discovery using these new data.
+scientists to make discovery using these new data. .
```

Comment:
Note that a stray "." characters was added.

Reply:
Yes, that was done to start the pull request. I will remove that.